tools/firely-server-import.rst

```diff
@@ -200,6 +200,7 @@ Known issues
 
 * FSI does not support scenarios where resources of different FHIR versions are stored in the same database;
 * When importing data from large ``*.ndjson`` files, the memory consumption may be quite high.
+* When importing STU3 resources, the field ``Patient.deceased`` will always be set to ``true`` if it exists. This is caused by an error in the FHIR STU3 specification. In case you would like to use FSI with STU3 resources, please contact us.
```

Comment:
Link to contact page for convenience please

Reply:
Added, thank you!